dtests/src/test/scala/io/snappydata/hydra/SnappyHydraRunner.scala

```diff
@@ -65,7 +65,11 @@ class SnappyHydraRunner extends SnappyTestRunner {
     val c14 = "grep -v java.lang.reflect.InvocationTargetException"
     val c15 = "grep -v org.apache.spark.storage.ShuffleBlockFetcherIterator." +
         "throwFetchFailedException"
-    val command1 = c1 #| c2 #| c3 #| c4 #| c5 #| c6 #| c7 #| c8 #| c12 #| c13 #| c14 #| c15
+    val c16 = "grep -v java.lang.IllegalStateException"
+    val c17 = "grep -v com.gemstone.gemfire.distributed.LockServiceDestroyedException"
+    val c18 = "grep -v com.gemstone.gemfire.GemFireIOException"
+    val command1 = c1 #| c2 #| c3 #| c4 #| c5 #| c6 #| c7 #| c8 #| c12 #| c13 #| c14 #| c15 #|
+        c16 #| c17 #| c18
```

Comment:
Are these blanket checks proper or should these be test specific? Also I think its much better that test itself should fail for unexpected exceptions rather than grep logs which is error prone.